scripts/joinmarket-qt.py

```diff
@@ -573,6 +580,17 @@ def initUI(self):
         innerTopLayout.addWidget(amountLabel, 4, 0)
         innerTopLayout.addWidget(self.amountInput, 4, 1, 1, 2)
 
+        changeLabel = QLabel('Change address')
```

Comment:
I think 'Custom change address' is a better label, albeit (a) that's a bit long, and (b) the tooltips already covers it.

My reason: it will definitely confuse people otherwise, even if only briefly, and people very often don't look at the tooltips. As for (a) we already have 'Number of counterparties' which is longer.

Reply:
You're right, and this is what I was trying to mitigate with the "(optional)" placeholder, although I admit there is still room for confusion: "Not all CoinJoins have change, but if I do a CoinJoin with change and don't specify a change address here, will my change get lost?"

I think "Custom change address" is too long. "Custom change" might work, although it leaves no indication (other than the tooltip) that they are supposed to put an address there.

Unrelated, but "Number of counterparties" might better be called "CoinJoin size" or something... in any case there's no need to add more ugliness :).

Reply:
I think you should call it 'Custom change address', and I don't want to change 'number of counterparties' either, I think clarity takes precedence over shortness (until and unless any users tell us different).

Reply:
If you insist...